### DISCUSSION TITLE
Date for releasing environment for deformable object manipulation

### DISCUSSION THREAD

**Discussion by ruyingdong** [isaac-sim/IsaacLab#16] [Q&A] [answered]
Hi! do you guys have exact date for releasing environment for deformable object manipulation? I have not found related code in the github yet. Will this come up in this month? Pls let me know, thank you very much.

**Comment by animesh-garg** [comment, marked as answer]
We don't have a precise date but in the order of a few weeks. 

Please see the answer to question #10 .

A version of tensorized cloth support now exists in Isaac Sim 2022.2.0 onwards. We are working on using this new API and integrating it with Orbit for the best experience and performance.

General deformable (soft) body support with TensorAPI is being built and is on the near-term development roadmap already.
